ydb/core/fq/libs/checkpointing/checkpoint_coordinator.cpp

```diff
@@ -103,6 +103,12 @@ void TCheckpointCoordinator::Handle(NYql::NDqs::TEvReadyState::TPtr& ev) {
         AllActorsSet.insert(actorId);
     }
 
+    CC_LOG_I("ActorsToTrigger count: " << ActorsToTrigger.size() << ", ActorsToNotify count: " << ActorsToNotify.size() << ", ActorsToWaitFor count: " << ActorsToWaitFor.size());
```

Comment:
1. Do we need to use info instead of debug in this case? And the same below.
2. If we have different sources in the same graph ("PgSource", "S3Source") will it work fine?

Reply:
1. Changed to debug
2. I think yes, S3Source just not added to ActorsToTrigger (not tested)